hooks/pre-commit

```diff
@@ -7,9 +7,9 @@ eval "$(git rev-parse --show-toplevel)/hooks/apply-git-secrets.sh"
 git secrets --pre_commit_hook -- "$@"
 
 CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
-if [ "$CURRENT_BRANCH" = "master" ]
+if [ "$CURRENT_BRANCH" = "main" ]
```

Comment:
I'm sure there will be many local `master` branches for a while, so let's keep both for now.  Maybe we could have a different error like "the default branch is now main"

Reply:
Done